repository/e.json

```
@@ -1439,6 +1439,17 @@
 				}
 			]
 		},
+		{
+			"name": "evscript",
+			"details": "https://github.com/ISSOtm/sublime-evscript",
+			"labels": ["language syntax", "game boy", "RGBDS"],
+			"releases": [
+				{
+					"sublime_text": ">=3092",
```

Syntax definition specifies `version: 2`, which requires at least Sublime Text 4.

The first stable release is:

```suggestion
					"sublime_text": ">=4107",
```

---

Oh, I wasn't aware of that. Since those are simple changes, I'd like to prioritise supporting earlier ST versions. Are there other caveats I should be aware of?

(And, for that matter, are there automated checks for this? I haven't found any.)

---

v2 applies meta scopes a bit different in detail.

Your syntax seems not to be affected by that change, so removing `version: 2` and reverting to `pop: true` should be all.

---

@ISSOtm indeed, supporting versions of ST much older than 4107 sounds like a wasted effort. Please apply that suggestion so we can continue the review.

---

Whoops, sorry, this fell off my radar... I pushed 1.0.1, which fixes deathaxe's review.

My apologies for the delay!